samples/RunablePallet.py

```diff
@@ -19,16 +19,14 @@ class RunablePallet(Pallet):
     def build(self, configuration):
         self.log.info('building pallet')
 
-    def lift(self):
-        self.log.info('lifting pallet')
-
     def ship(self):
         self.log.info('shipping pallet')
 
+        self.send_email('stdavis@utah.gov', 'test subject', 'hello body')
```

Comment:
do you want your email here?

Reply:
Hmm. Both of ours are in `config.py` as well. Maybe I'll make them more generic...